samples/Calling/src/app/utils/AppUtils.ts

```diff
@@ -9,7 +9,7 @@ import { v1 as generateGUID } from 'uuid';
  */
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export const fetchTokenResponse = async (): Promise<any> => {
-  const response = await fetch('/token');
+  const response = await fetch('/token?scope=voip');
```

Comment:
Question about the PR title, this is a client-side change right? And why are we removing scope from client side request?

Reply:
Maybe like [Sample/Chat&Call] adding scope param for token request

Reply:
Then add the related PR of server side in the description, so people can check then together when reviewing

Reply:
Updated :)